rcli/display.py

```diff
@@ -106,7 +106,7 @@ def print_status(msg, color):
             raise e.exc  # pylint: disable=raising-bad-type
     except (KeyboardInterrupt, EOFError):
         raise
-    except:
+    except Exception:
```

Comment:
Do we actually care about them being subclasses of `Exception`? We already have the two major `BaseException` subclasses above; does this cut anything out?

Reply:
I don't care, but pylint does.   Is it better to `except Exception:` or to `except:  # Ignore such-and-such`?

Reply:
You generally still need the broad-except ignore for `except Exception`.

Reply:
My local tests don't indicate this.